thumb2_disasm/il_thumb2.cpp

```diff
@@ -948,14 +948,14 @@ bool GetLowLevelILForThumbInstruction(Architecture* arch, LowLevelILFunction& il
 		il.AddInstruction(WriteArithOperand(il, instr, il.DivSigned(4, ReadArithOperand(il, instr, 0), ReadArithOperand(il, instr, 1))));
 		break;
 	case armv7::ARMV7_SBC:
-		il.AddInstruction(WriteILOperand(il, instr, 0, il.SubBorrow(4, ReadILOperand(il, instr, 1),
-									       ReadShiftedOperand(il, instr, 2),
+		il.AddInstruction(WriteArithOperand(il, instr, il.SubBorrow(4, ReadArithOperand(il, instr, 0),
```

Comment:
Oh, also fixed lifting for the 2-operand version here. It was just lifting an `unimplemented` third operand before.